src/web/components/Content/Content.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Content from './';
+
+describe('Container', () => {
+  let content;
+  let wrapper;
+
+  beforeEach(() => {
+    content = <h1>Hello world</h1>;
+    wrapper = shallow(<Content>{content}</Content>);
+  });
+
+  it('should render children component', () => {
+    expect(wrapper.children()).toHaveLength(1);
+  });
+
+  it('should render a content without size by default', () => {
+    expect(wrapper.find('div.content').prop('className')).toBe('content');
```

Comment:
class `content` is something from Bulma ?

Reply:
yes, content is css class from bulma. It's a kind of container for formatting text contents such as lists, titles, paragraphs, etc.
https://bulma.io/documentation/elements/content/